src/test/java/core/basesyntax/service/impl/WriterImplTest.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.exception.FruitShopException;
+import core.basesyntax.service.Writer;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class WriterImplTest {
+    private static final String INCORRECT_PATH_OF_OUTPUT_FILE
+            = "src/test/service/resources/output.csv";
```

Comment:
```suggestion
            = "incorrect/path/to/file/output.csv";
```
Make it immediately obvious